docs/en/deploy/Running-Alluxio-On-a-HA-Cluster.md

```diff
@@ -333,18 +333,50 @@ In order to add a master, the Alluxio cluster must operate in HA mode. If you ar
 a single master cluster, you must configure it to be an HA cluster before having more than one master.
 
 #### Alluxio HA cluster with embedded journal
+When internal leader election is used, Alluxio masters are determined with a quorum. Adding an removing
+masters requires to keep this quorum in a consistent state. 
 
-When internal leader election is used, Alluxio masters are determined. Adding or removing
-master nodes requires:
+##### Adding a new master
+In order to prevent inconsistencies in the cluster configuration across masters, only a single master
+should be added to an existing embedded journal cluster. 
```

Comment:
Only one master can be added or only one master at a time (adding multiple masters sequentially)

Reply:
I'm not sure which one is appropriate for a tutorial.
@apc999 ?

Reply:
The current way is fine, we don't want users to continually adding new masters. If they do, they should following the whole procedure of `Adding a new master` section

Reply:
SG